<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/images/felt.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_00">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="Card" stdset="0">
     <string>object.Card = label00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_62">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_64">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_66">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stock_previous">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stock">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(0, 170, 255); 
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stock_current">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="smileButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click me :) </string>
    </property>
   </widget>
   <zorder>label_00</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_20</zorder>
   <zorder>label_21</zorder>
   <zorder>label_22</zorder>
   <zorder>label_30</zorder>
   <zorder>label_31</zorder>
   <zorder>label_32</zorder>
   <zorder>label_33</zorder>
   <zorder>label_40</zorder>
   <zorder>label_41</zorder>
   <zorder>label_42</zorder>
   <zorder>label_43</zorder>
   <zorder>label_44</zorder>
   <zorder>label_50</zorder>
   <zorder>label_51</zorder>
   <zorder>label_52</zorder>
   <zorder>label_53</zorder>
   <zorder>label_54</zorder>
   <zorder>label_55</zorder>
   <zorder>label_60</zorder>
   <zorder>label_61</zorder>
   <zorder>label_62</zorder>
   <zorder>label_63</zorder>
   <zorder>label_64</zorder>
   <zorder>label_65</zorder>
   <zorder>label_66</zorder>
   <zorder>stock_previous</zorder>
   <zorder>stock</zorder>
   <zorder>smileButton</zorder>
   <zorder>stock_current</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
